keyboards/rephlex/2k/config.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2023 RephlexZero (@RephlexZero)
+SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+
+#ifdef VIA_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 1
+#endif
+
+#define EECONFIG_KB_DATA_SIZE 6
+
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+#define DEBOUNCE 0
+/*#define DEBUG_MATRIX*/
+#define DEBUG_MATRIX_SCAN_RATE
+
+#define ADC_RESOLUTION 12
+
+#define CALIBRATION_RANGE 255
+
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 2
+#define MATRIX_PINS {{ GP26, GP27 }}
```

Comment:
Shouldn't be needed here, if direct pins is set in info.json.
```suggestion
```

Reply:
I am unsure how to utilise direct_pins in matrix.c, compiler throws errors that it isn't defined and I couldn't figure it out from looking at quantum

Reply:
Solved in [2b77928](https://github.com/qmk/qmk_firmware/pull/21715/commits/2b77928a816bfbcd90903e4bd27d037b1986e8ce)